core/sass/euexit-form.scss

```diff
@@ -0,0 +1,30 @@
+@import './partials/base';
+@import './components/form-wizard';
+@import './components/breadcrumbs';
+
+#content { 
+  h1 {
+    margin-bottom: 20px;
+  }
+  .body-text {
+    margin-bottom: 50px;
+  }
+  .breadcrumbs {
+    border-bottom: 0;
+    padding-top: 0;
+  }
+  .form-label {
+    margin-bottom: 10px;
```

Comment:
if ur going to add margin to the bottom of form labels you should also add it to `form-hint` as well to be consistent